ksqldb-common/src/main/java/io/confluent/ksql/metrics/ConsumerCollector.java

```diff
@@ -196,4 +206,20 @@ public double aggregateStat(final String name, final boolean isError) {
   public String toString() {
     return getClass().getSimpleName() + " id:" + this.id + " " + topicSensors.keySet();
   }
+
+  private static Sensor configureTotalBytesSum(final Metrics metrics) {
+    final String description = "The total number of bytes consumed across all consumers";
+    final Sensor sensor = metrics.sensor(CONSUMER_ALL_TOTAL_BYTES_SUM);
+    sensor.add(
+        metrics.metricName(
+            CONSUMER_ALL_TOTAL_BYTES_SUM,
+            CONSUMER_COLLECTOR_METRICS_GROUP_NAME,
+            description),
+        new CumulativeSum());
+    return sensor;
+  }
+
+  protected Metrics getMetrics() {
```

Comment:
This function seems not used? Only MetricsCollectors.metrics() is needed elsewhere.

Reply:
I'll remove, this was used in a previous iteration of the PR